README.md

```diff
@@ -38,18 +38,18 @@ import { tracked } from '@glimmer/tracking';
 import { compileJS } from 'ember-repl';
 
 export class Renderer extends Component {
-  @tracked myComponent;
+  @tracked repl;
```

Comment:
Can we call this compileResult?

REPL implies more than what these functions do.
I keep forgetting what REPL is, so I just looked it up.

Read
Eval
Print 
Loop

These functions are the 'eval' part, but the main application of these functions at runtime is for broader applications that, as a whole, are a REPL.

Reply:
You could even paste this into the README: https://github.com/NullVoxPopuli/ember-repl/blob/main/addon/types.ts#L1
Which is public API via: https://github.com/NullVoxPopuli/ember-repl/blob/main/addon/index.ts#L6

Reply:
Oh nice. That sounds like a right thing to do.